src/ticketing/controller/tickets.controller.ts

```diff
@@ -99,4 +102,15 @@ export class TicketsController {
 
         return ticketsOfIdentity;
     }
+
+    @Cron('* * 0 * * *')
+    async invalidTickets(): Promise<void> {
+        this.ticketsService.invalidTickets();
+    }
+
+    @HttpCode(200)
```

Comment:
```suggestion
    @HttpCode(204)
```
2xx mit 204 NoContent wäre besser.